packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -929,14 +929,25 @@ function testUrlMatchCookie(cookie: Cookie, url: URL): boolean {
   return testUrlMatchCookiePath(cookie, normalizedUrl);
 }
 
-export function bidiToPuppeteerCookie(bidiCookie: Bidi.Network.Cookie): Cookie {
+export function bidiToPuppeteerCookie(
```

Comment:
Non blocking: Should we move these out of Page an into separate file to make it clear that they are utility now that we are exporting them?

Reply:
I think we can move them completely to BrowserContext once we remove the page-level cookies API.